### PR TITLE
Docs for Opentelemetry on the Cloud operator

### PR DESCRIPTION
**Pull request by ignasi35**
refs https://github.com/lightbend/akka-platform-meta/issues/179

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/operator-reference.adoc [hunk 1]
```diff
@@ -281,7 +281,14 @@ Additionally to Up and Non-Up, there is also Unreachable. This is the number of
 
 These numbers can be useful to watch the Akka cluster state during rolling updates to make sure it is in a normal state. See xref:troubleshooting.adoc[troubleshooting deployment status] for some examples.
 
-== Reporting Metrics to Prometheus
+== Telemetry
+
+The Akka Operator supports several modes of telemetry reporting. First, there's the classic architecture where the application directly pushes traces or exposes a prometheus port. Second, is the architecture based on the OpenTelemetry side-car.
```
Comment (marcospereira) [inline docs-source/docs/modules/deployment/pages/operator-reference.adoc:286]:
```suggestion
The Akka Operator supports several modes of telemetry reporting. First, there's the classic architecture where the application directly pushes traces or exposes a Prometheus port. Second, is the architecture based on the OpenTelemetry side-car.
```

Comment (marcospereira) [reply docs-source/docs/modules/deployment/pages/operator-reference.adoc:286]: By `side-car` you mean https://github.com/open-telemetry/opentelemetry-collector? If so, let call use `OpenTelemetry Collector` then.

**Review by marcospereira — COMMENTED** [review]
Minor comments.

I wonder if we can somehow use https://github.com/open-telemetry/opentelemetry-operator here, or if that would make things more complex (or less stable). WDYT?

Also, is it Jaeger, Jagger, Jaegger, Jager? 😅

**Comment by ignasi35** [comment]
There's a handful of technical blockers to complete the implementation side of this feature at this time. I'll close this PR and we can revisit it in the future.

**Pull request closed without merging**